examples/custom_plotting.ipynb

```diff
@@ -0,0 +1,377 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Customizing contact map plots\n",
+    "\n",
+    "The main plotting methods use matplotlib. If you're already familiar with matplotlib, you'll have a good background already, and if not, what you learn about Contact Map Explorer will transfer to other projects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "import mdtraj as md\n",
+    "traj = md.load(\"5550217/kras.xtc\", top=\"5550217/kras.pdb\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from contact_map import ContactFrequency\n",
+    "traj_contacts = ContactFrequency(traj)\n",
+    "frame_contacts = ContactFrequency(traj[0])\n",
+    "diff = traj_contacts - frame_contacts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Putting multiple plots in one figure"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You may want to plot multiple maps on one figure. This can easily be done by using the `plot_axes` method, which acts just like `plot` except that it requires that you give it a Matplotlib `Axes` object to plot into."
```

Comment:
isn't "in" more correct here?
```suggestion
    "You may want to plot multiple maps in one figure. This can easily be done by using the `plot_axes` method, which acts just like `plot` except that it requires that you give it a Matplotlib `Axes` object to plot into."
```